components/atom/backToTop/UXDEF.md

```diff
@@ -0,0 +1,126 @@
+# SUI - Back to Top
+A simple way to return back to the top of a long content page.
+
+|   Status          | WIP |
+|   :----           |   :---- |
+|   Current version |   1.0|
+|   Category        |   Atom |
+|   Owners          |   UX @Irene L, @Chris J |
```

Comment:
Could be great to use real links to people if they have access to GitHub 😉

Reply:
un "+1" total a esto, a implementar una vez all Designers tengan gitHub (y yo sepa los users claro)